src/writer.rs

```diff
@@ -32,10 +31,15 @@ pub struct Writer<'a, W> {
 }
 
 impl<'a, W: Write> Writer<'a, W> {
+    /// Creates a `Writer` given a `Schema` and something implementing the `Write` trait to write
```

Comment:
nit: do you think we should be more explicit and specify `io::Write`?
It seems almost obvious that we don't mean, say `fmt::Write`, but one never knows :)

Reply:
It doesn't hurt